card-detection-api/card-detection-api/Models/Detection.cs

```diff
@@ -44,6 +42,50 @@ public override string ToString()
         {
             return $"{Card} {Confidence * 100}% at [{BoundingBox}]";
         }
+
+         public static List<Detection> filterOverlaps(List<Detection> detections)
+        {
+            for(int i = 0; i < detections.Count; i++)
+            {
+                for(int j = i+1; j<detections.Count; j++)
+                {
+                    if (FindOverlap(detections[i], detections[j]) == null) 
+                    {continue;}
```

Comment:
Formatting is niet zoals code style
```suggestion
                    if (FindOverlap(detections[i], detections[j]) == null) 
                        continue;
```